Midichlorians/Dev/Beta2/MidichloriansDataDictionary.h

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2015 Maxim Zhurovich <zhurovich@gmail.com>
+          (c) 2015 Dmitry "Dima" Korolev <dmitry.korolev@gmail.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*******************************************************************************/
+
+#ifndef __cplusplus
+// If you get this error building an Objective-C project, #include neither this file nor "MidichloriansImpl.h".
+// Use "Midichlorians.h" instead.
+// It wraps the inner, platform-independent, event tracking code, and allows client code to stay `.m`.
+#error "This C++ header should be `#include`-d or `#import`-ed from an `.mm`, not an `.m` source file."
+#endif
+
+#ifndef CURRENT_MIDICHLORIANS_DATA_DICTIONARY_H
```

Comment:
I was thinking of retiring, but existing CTFO users will continue to use the old version. Should we support both in our new server?